docs/en/using-status/leave-a-group-chat.md

```diff
@@ -8,7 +8,31 @@ hide:
 
 # Leave a group chat
 
- :octicons-tools-24:
+If you don’t want to be a member of a [group chat][understand-group-chats] anymore, you can leave the chat. You can always [re-join the chat][join-a-group-chat] later if you change your mind.
 
-!!! note ""
-     We're working on this content.
+Once you leave, a note appears in the group chat's message history letting other members know that you left the chat. You can still see the previous message history for a group chat you left.
```

Comment:
> Once you leave, a note appears in the group chat's message history letting other members know that you left the chat.

This needs a screenshot to show the user what s/he can expect others to see. Could you please add the screenshot to [the Notion list](https://www.notion.so/Simplified-screenshots-in-Status-Help-9b99304babb042e2a63f4bc94aa0b179)?

Reply:
Done! I created [an issue](https://github.com/status-im/help.status.im/issues/396) for us to remember about this once the design team finishes the screen.